### PR TITLE
[AND-2437]MoPub adapter play MREC audio should be muted by default.

### PR DESCRIPTION
**Pull request by HuFangshuai**
This PR have been created for developing ticket https://vungle.atlassian.net/browse/AND-2437 for MoPub adapter 6.5.x

### CODE REVIEW

**Review by palrahul — COMMENTED** [review]
@HuFangshuai 
1) we should have a way to allow Publishers set an key/value for LocalExtra for banner request similar to how its done for Interstitial https://github.com/Vungle/mopub-android-mediation/blob/6307ee8b55ffa4e71b5f1a292097872ca54568e8/Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java#L116
2) Even though we currently do not support Video/Audio for Vungle banners , it might be a good idea to not limit the getting of audio setting override from Publisher and pass it to SDK AdConfig ( seems currently we have that logic only for MREC) That way if and when in future we decide for any reason to roll in a thumbnail video, we wont have to come back and make the changes. On SDK side we pass this value through MRAID bridge for all MRAID ads irrespective of the size/format

**Comment by HuFangshuai** [comment]
> @HuFangshuai
> 
> 1. we should have a way to allow Publishers set an key/value for LocalExtra for banner request similar to how its done for Interstitial https://github.com/Vungle/mopub-android-mediation/blob/6307ee8b55ffa4e71b5f1a292097872ca54568e8/Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java#L116
> 2. Even though we currently do not support Video/Audio for Vungle banners , it might be a good idea to not limit the getting of audio setting override from Publisher and pass it to SDK AdConfig ( seems currently we have that logic only for MREC) That way if and when in future we decide for any reason to roll in a thumbnail video, we wont have to come back and make the changes. On SDK side we pass this value through MRAID bridge for all MRAID ads irrespective of the size/format

@palrahul good suggestion. I have fixed for both MREC and Banner ads.

File: Vungle/src/main/java/com/mopub/mobileads/VungleExtrasBuilder.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        return extrasMap;
+    }
+
+    public static void adConfigWithLocalExtras(AdConfig adConfig, Map<String, Object> localExtras) {
+        if (localExtras.containsKey(EXTRA_START_MUTED)) {
+            Object isStartMuted = localExtras.get(EXTRA_START_MUTED);
+            if (isStartMuted instanceof Boolean)
+                adConfig.setMuted((Boolean) isStartMuted);
+        } else {
+            Object isSoundEnabled = localExtras.get(EXTRA_SOUND_ENABLED);
```
Comment (HuFangshuai) [inline Vungle/src/main/java/com/mopub/mobileads/VungleExtrasBuilder.java:52]: We keep this original SOUND_ENABLED key at least for 1 release

File: Vungle/src/main/java/com/mopub/mobileads/VungleExtrasBuilder.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A helper class for creating a network extras map that can be passed to the adapter to make
+ * customizations specific to Vungle.
+ */
+public final class VungleExtrasBuilder {
+
+    private static final String EXTRA_START_MUTED = "startMuted";
```
Comment (alexander-gubov) [inline Vungle/src/main/java/com/mopub/mobileads/VungleExtrasBuilder.java:14]: we already have the same in VungleMediationSettings

Comment (HuFangshuai) [reply Vungle/src/main/java/com/mopub/mobileads/VungleExtrasBuilder.java:14]: @alexander-gubov Currently we are moving ad settings from app level to placement level, so some of them are duplicates. It's a good suggestion, May refactor these common settings in one super class in future. I backlog this issue.
cc: @palrahul

**Pull request merged**